__tests__/app.test.js

```diff
@@ -165,4 +174,34 @@ describe('App', ()=>{
             .expect(400); 
         });
     })
+    describe('POST /api/articles/:article_id/comments', () => {
+        test('POST 201: should add a comment to an article', () => {
+            return request(app)
+            .post('/api/articles/1/comments')
+            .send({username: 'lurker', body: 'test body'})
+            .expect(201)
+            .then(({body}) => {
+                expect(body.comment).toMatchObject({
+                    article_id: 1,
+                    author: 'lurker',
+                    body: 'test body',
+                    comment_id: 19,
+                    created_at: expect.any(String),
+                    votes: 0
+                })
+            })
+        });
+        test('POST 400: should return 400 if no username provided', () => {
+            return request(app)
+            .post('/api/articles/1/comments')
+            .send({body: 'test body'})
+            .expect(400)
+        });
+        test('POST 400: should return 400 if no body provided', () => {
```

Comment:
Nice testing so far, however, we should look to also add the following...
Status 201, ignores unnecessary properties
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 404 if username does not exist